### PR TITLE
DOCSP-14426 whats new

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-14426

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/bd77764/node/docsworker-xlarge/DOCSP-14426-WhatsNew/whats-new/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+* :ref:`Version 4.0 <version-4.0>`
+* :ref:`Version 3.6 <version-3.6>`
+
+.. _version-4.0:
+
+What's New in 4.0
+-----------------
+
+New features of the 4.0 Node.js driver release include:
```
Comment (terakilobyte) [inline source/whats-new.txt:23]: This feels incomplete. Is this meant to target the master branch and 4.0 or just the 3.6 branch?

File: source/whats-new.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+.. _version-3.6:
+
+What's New in 3.6
+-----------------
+
+New features of the 3.6 Node.js driver release include:
+
+- Added support for the MONGODB-AWS authentication mechanism using Amazon Web Services (AWS) Identity and Access Management (IAM) credentials
```
Comment (terakilobyte) [inline source/whats-new.txt:32]: If we have these topics covered in the documentation we should link to them.

**Review by terakilobyte — CHANGES_REQUESTED** [review]
a few comments/suggestions

File: source/fundamentals/authentication/mechanisms.txt [hunk 3]
```diff
@@ -122,6 +122,7 @@ in the following sample code.
    SCRAM </release-notes/3.0-scram/>`, any ``MONGODB-CR`` user
    authentication requests fail.
 
+.. _mongosb-aws:
```
Comment (terakilobyte) [inline source/fundamentals/authentication/mechanisms.txt:125]:
```suggestion
.. _mongodb-aws: 
```

**Review by terakilobyte — APPROVED** [review]
lgtm with one correction

File: source/whats-new.txt [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+New features of the 4.0 Node.js driver release include:
+
+.. _version-3.6:
+
+What's New in 3.6
+-----------------
+
+New features of the 3.6 Node.js driver release include:
+
+- Added support for the :ref:`MONGODB-AWS <mongosb-aws>` authentication mechanism using Amazon Web Services (AWS) Identity and Access Management (IAM) credentials
```
Comment (terakilobyte) [inline source/whats-new.txt:32]:
```suggestion
- Added support for the :ref:`MONGODB-AWS <mongodb-aws>` authentication mechanism using Amazon Web Services (AWS) Identity and Access Management (IAM) credentials
```

**Comment by kyuan-mongodb** [comment]
Hi @emadum, This PR is ready for review

File: source/whats-new.txt [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- Added support for Online Certificate Status Protocol (OCSP)
+- The :node-api:`find()<Collection.html#find>` method supports ``allowDiskUse()`` for sorts that require too much memory to execute in RAM
+- The :ref:`update() <updateDocuments>` and :ref:`reaplceOne() <replacementDocument>` methods support index hints
```
Comment (emadum) [inline source/whats-new.txt:35]:
```suggestion
- The :ref:`update() <updateDocuments>` and :ref:`replaceOne() <replacementDocument>` methods support index hints
```

**Review by emadum — CHANGES_REQUESTED** [review]
Found a typo but otherwise LGTM 👍

**Review by emadum — APPROVED** [review]

**Pull request merged**